app/templates/summary.html

```diff
@@ -22,13 +22,27 @@ <h2 class="header__subtitle mars">Please check carefully before submission.</h2>
 
       {% for item in block.items %}
 
+      {% for answer in item.state_answers %}
+
         {% set id = summary_id ~ "-" ~ loop.index0 %}
 
-        <dt class="summary__question summary__question--{{item.type}}" id="{{id}}">{{item.question | safe}}</dt>
+        <dt class="summary__question summary__question--{{item.type}}" id="{{id}}">
+          {% if item.state_answers|length == 1 %}
```

Comment:
I think you're right, @hamishtaplin also mentioned the same thing. I'm currently trying to figure out how 'answers' and 'items' fit in as 'SummaryItems'. Any suggestions are welcomed!